src/main/scala/is/hail/methods/SplitMulti.scala

```diff
@@ -87,23 +87,23 @@ class SplitMultiPartitionContextIR(
   private val locusType = matrixType.rowType.fieldByName("locus").typ
   val f = rowF()
 
-  def constructSplitRow(splitVariants: Iterator[(Variant, Int)], rv: RegionValue, wasSplit: Boolean): Iterator[RegionValue] = {
+  def constructSplitRow(splitVariants: Iterator[(Locus, IndexedSeq[String], Int)], rv: RegionValue, wasSplit: Boolean): Iterator[RegionValue] = {
     if (!globalsCopied)
       copyGlobals()
     splitRegion.clear(globalsEnd)
     rvb.start(matrixType.rvRowType)
     rvb.addRegionValue(matrixType.rvRowType, rv)
     val oldRow = rvb.end()
     val oldEnd = splitRegion.size
-    splitVariants.map { case (sjv, aIndex) =>
+    splitVariants.map { case (svjLocus, svjAlleles, aIndex) =>
```

Comment:
`sv` stands for split variant, right? I don't fine `j` helpful for readability here or in 221 below. And on line 130 you use `newLocus` and `newAlleles` for the same concept. (not in scope, but I'd also change `splitrv` to `splitRV`)